mods/ts/rules/defaults.yaml

```diff
@@ -376,84 +373,40 @@
 		InitialDelay: 0
 		CloakDelay: 90
 	MustBeDestroyed:
+	RenderSprites:
 
 ^Tank:
-	Inherits: ^GainsExperience
-	AppearsOnRadar:
+	Inherits: ^Vehicle
 	Mobile:
-		Crushes: wall, crate
+		Crushes: wall, crate, infantry
 		TerrainSpeeds:
-			Clear: 90
+			Clear: 70
 			Road: 100
-			Rail: 80
+			Rail: 100
 			DirtRoad: 90
-			Rough: 70
-			Tiberium: 80
-			BlueTiberium: 80
-			Veins: 80
-		ROT: 5
-		Voice: Move
-	Selectable:
-	SelectionDecorations:
-		Palette: pips
-	Voiced:
-		VoiceSet: Vehicle
-	TargetableUnit:
-		TargetTypes: Ground, Vehicle
-	Repairable:
-		RepairBuildings: gadept
-		Voice: Move
-	Passenger:
-		CargoType: Vehicle
-		Voice: Move
-	AttackMove:
-		Voice: Move
-	HiddenUnderFog:
-	GivesExperience:
-	DrawLineToTarget:
-	ActorLostNotification:
-	ProximityCaptor:
-		Types: Vehicle
-	Capturable:
-		Type: Vehicle
-		CaptureThreshold: 1
-		CancelActivity: True
-	GivesBounty:
-	UpdatesPlayerStatistics:
-	CombatDebugOverlay:
-	Guard:
-		Voice: Move
-	Guardable:
-	BodyOrientation:
-		CameraPitch: 90
-	Huntable:
-	ScriptTriggers:
-	Explodes:
-		Weapon: UnitExplodeSmall
-		EmptyWeapon: UnitExplodeSmall
-	UpgradeManager:
-	UpgradeOverlay@EMPDISABLE:
-		UpgradeTypes: empdisable
-		UpgradeMinEnabledLevel: 1
-		UpgradeMaxAcceptedLevel: 2
-		Palette: disabled
-	DisableUpgrade@EMPDISABLE:
-		UpgradeTypes: empdisable
-		UpgradeMinEnabledLevel: 1
-		UpgradeMaxAcceptedLevel: 2
-	TimedUpgradeBar@EMPDISABLE:
-		Upgrade: empdisable
-		Color: 255,255,255
-	Cloak@CLOAKGENERATOR:
-		UpgradeTypes: cloakgenerator
-		UpgradeMinEnabledLevel: 1
-		UpgradeMaxAcceptedLevel: 2
-		InitialDelay: 0
-		CloakDelay: 90
-	MustBeDestroyed:
+			Rough: 60
+			Tiberium: 70
+			BlueTiberium: 70
+			Veins: 70
+
+^VoxelVehicle:
+	Inherits: ^Vehicle
+	RenderVoxels:
+	WithVoxelBody:
+
+^VoxelTank:
+	Inherits: ^Tank
+	RenderVoxels:
+	WithVoxelBody:
+
+^CivilianVoxelVehicle:
+	Inherits: ^VoxelVehicle
+	-RenderSprites: # They don't have icons so this is needed to satisfy --check-yaml
```

Comment:
I vote for avoiding comments in YAML wherever possible, because the utility doesn't like them and keeps removing them. Also I don't think this is exactly needed here.